packages/xstate-solid/rollup.config.js

```diff
@@ -0,0 +1,52 @@
+import { terser } from 'rollup-plugin-terser';
+import typescript from 'rollup-plugin-typescript2';
+import pkg from './package.json';
+
+const makeExternalPredicate = (externalArr) => {
+  if (externalArr.length === 0) {
+    return () => false;
+  }
+  const pattern = new RegExp(`^(${externalArr.join('|')})($|/)`);
+  return (id) => pattern.test(id);
+};
+
+function createUmdConfig({ input, output: file, name }) {
+  return {
+    input,
+    external: makeExternalPredicate(Object.keys(pkg.peerDependencies)),
+    output: {
+      file,
+      format: 'umd',
+      name,
+      globals: {
+        xstate: 'XState',
+        '@xstate/fsm': 'XStateFSM',
+        'solid-js': 'Solid',
+        'solid-js/store': 'SolidStore',
+      }
+    },
+    plugins: [typescript(), terser({ include: [/^.+\.min\.js$/] })]
```

Comment:
I think this is redundant, although I could be wrong. I have never seen this option being used (although now I see that some of our packages use this).
```suggestion
    plugins: [typescript(), terser()]
```